repos/system_upgrade/el7toel8/actors/inhibitwhenluks/actor.py

```diff
@@ -21,12 +21,16 @@ def process(self):
         for storage_info in self.consume(StorageInfo):
             for blk in storage_info.lsblk:
                 if blk.tp == 'crypt':
-                    report_with_remediation(
-                        title='LUKS encrypted partition detected',
-                        summary='Upgrading system with encrypted partitions is not supported',
-                        remediation='If the encrypted partition is not system one and the system '
-                                    'is not depending on it, you can remove/blacklist it from '
-                                    'the system',
-                        severity='high',
-                        flags=['inhibitor'])
+                    create_report([
+                        reporting.Title('LUKS encrypted partition detected'),
+                        reporting.Summary('Upgrading system with encrypted partitions is not supported'),
+                        reporting.Severity(reporting.Severity.HIGH),
+                        reporting.Tags([reporting.Tags.BOOT, reporting.Tags.ENCRYPTION]),
+                        reporting.Flags([reporting.Flags.INHIBITOR]),
+                        reporting.Remediation(
```

Comment:
I'd get rid of this hint, it doesn't seem to provide any info at all.

Reply:
fair enough